tmva/sofie/src/RModel.cxx

```diff
@@ -538,6 +546,10 @@ namespace SOFIE{
             fGC += "      fTensor_" + i.first + " = *reinterpret_cast<std::vector<int64_t>*>(rootFile->Get(\"";
             fGC += dirName + "/" + tensor_name + "\"));\n";
          }
+         else if (i.second.fType == ETensorType::BOOL) {
+            fGC += "      fTensor_" + i.first + " = *reinterpret_cast<std::vector<bool>*>(rootFile->Get(\"";
+            fGC += dirName + "/" + tensor_name + "\"));\n";
+         }
```

Comment:
I did not understand the use of this code. I guess Equal operator has no initialized tensors?